.github/CODEOWNERS

```diff
@@ -4,7 +4,7 @@
 # More info: https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-on-github/about-code-owners
```

Comment:
I guess the language in Contributing is technically accurate:

https://github.com/ni/nimble/blob/2426ac9581dc104ebc0313c28d64b429f9c5e8a0/CONTRIBUTING.md?plain=1#L128-L130

Should we emphasize something like:

"Each file in a Pull Request requires approval from one of the code owners assigned to those files". Owners for different files are listed in CODEOWNER."

Reply:
Good idea, I clarified our policy and also mentioned that we usually wait for all reviewers even if it's not technically required.